mne/artifact_detection.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov 19 09:40:37 2019
+
+@author: an512
+"""
+
+from scipy.stats import zscore
+import mne
```

Comment:
mne import should come after numpy and scipy

Reply:
Actually you shouldn't `import mne` at all unless you're writing a tutorial or a test. You should do relative imports.